concepticondata/conceptlists/Snodgrass-1980-260.tsv

```diff
@@ -0,0 +1,261 @@
+Snodgrass-1980-260-78	78	474	DRESS	Dress
+Snodgrass-1980-260-79	79			Dresser
+Snodgrass-1980-260-80	80	908	DRUM	Drum
+Snodgrass-1980-260-81	81	1360	DUCK	Duck
+Snodgrass-1980-260-82	82	1905	EAGLE	Eagle
+Snodgrass-1980-260-83	83	1247	EAR	Ear
+Snodgrass-1980-260-84	84	1290	ELEPHANT	Elephant
+Snodgrass-1980-260-85	85	3088	ENVELOPE	Envelope
+Snodgrass-1980-260-86	86	1248	EYE	Eye
+Snodgrass-1980-260-87	87	1690	FENCE	Fence
+Snodgrass-1980-260-88	88	1303	FINGER	Finger
+Snodgrass-1980-260-89	89	227	FISH	Fish
+Snodgrass-1980-260-90	90	2623	FLAG	Flag
+Snodgrass-1980-260-91	91	239	FLOWER	Flower
+Snodgrass-1980-260-92	92	777	FLUTE	Flute
+Snodgrass-1980-260-93	93	1504	FLY (INSECT)	Fly
+Snodgrass-1980-260-94	94	1301	FOOT	Foot
+Snodgrass-1980-260-95	95	1845	BALL	Football
+Snodgrass-1980-260-96	96			Football helmet
+Snodgrass-1980-260-97	97	1302	FORK	Fork
+Snodgrass-1980-260-98	98	1312	FOX	Fox
+Snodgrass-1980-260-99	99	939	HORN (INSTRUMENT)	French horn
+Snodgrass-1980-260-100	100	503	FROG	Frog
+Snodgrass-1980-260-101	101	31	PAN	Frying pan
+Snodgrass-1980-260-102	102			Garbage can
+Snodgrass-1980-260-103	103	3089	GIRAFFE	Giraffe
+Snodgrass-1980-260-104	104	476	CUP	Glass 
+Snodgrass-1980-260-105	105	1898	SPECTACLES	Glasses
+Snodgrass-1980-260-106	106	867	GLOVE	Glove 
+Snodgrass-1980-260-107	107	1502	GOAT	Goat
+Snodgrass-1980-260-108	108	3144	GORILLA	Gorilla
+Snodgrass-1980-260-109	109	920	GRAPE	Grapes
+Snodgrass-1980-260-110	110	607	GRASSHOPPER	Grasshopper
+Snodgrass-1980-260-111	111	3090	GUITAR	Guitar
+Snodgrass-1980-260-112	112	1566	GUN	Gun
+Snodgrass-1980-260-113	113	2648	HAIR (HEAD)	Hair
+Snodgrass-1980-260-114	114	1481	HAMMER	Hammer
+Snodgrass-1980-260-115	115	1277	HAND	Hand
+Snodgrass-1980-260-116	116	770	EARRING	Hanger
+Snodgrass-1980-260-117	117	2597	HARP	Harp
+Snodgrass-1980-260-118	118	771	HAT	Hat
+Snodgrass-1980-260-119	119	1223	HEART	Heart
+Snodgrass-1980-260-120	120	2580	HELICOPTER	Helicopter
+Snodgrass-1980-260-121	121	615	HORSE	Horse
+Snodgrass-1980-260-122	122	1252	HOUSE	House
+Snodgrass-1980-260-123	123			Iron
+Snodgrass-1980-260-124	124			Ironing Board
+Snodgrass-1980-260-125	125	2770	JACKET	Jacket
+Snodgrass-1980-260-126	126	164	KANGAROO	Kangaroo
+Snodgrass-1980-260-127	127	854	KETTLE	Kettle
+Snodgrass-1980-260-128	128	1472	KEY	Key
+Snodgrass-1980-260-129	129	307	HAWK	Kite
+Snodgrass-1980-260-130	130	1352	KNIFE	Knife
+Snodgrass-1980-260-131	131	1464	LADDER	Ladder
+Snodgrass-1980-260-132	132	625	LAMP	Lamp
+Snodgrass-1980-260-133	133	628	LEAF	Leaf
+Snodgrass-1980-260-134	134	1297	LEG	Leg
+Snodgrass-1980-260-135	135	3091	LEMON	Lemon
+Snodgrass-1980-260-136	136	1139	LEOPARD	Leopard
+Snodgrass-1980-260-137	137	3729	LETTUCE	Lettuce
+Snodgrass-1980-260-138	138			Light bulb
+Snodgrass-1980-260-139	139			Light switch
+Snodgrass-1980-260-140	140	1386	LION	Lion
+Snodgrass-1980-260-141	141	478	LIP	Lips
+Snodgrass-1980-260-142	142	2397	LOBSTER	Lobster
+Snodgrass-1980-260-143	143	1596	LOCK	Lock
+Snodgrass-1980-260-144	144	867	GLOVE	Mitten 
```

Comment:
This is a specific type of glove that we would call Fäustling in German, right? So I suggest to unmap, since we already have *Glove* above. So we leave it unmapped for now. Here, the images come in handy, as they tell us what we are dealing with.

Reply:
okay, I will unmap this one. :)

Reply:
done